samples/Hello_Netherite_with_DotNetCore/HelloDF.csproj

```diff
@@ -5,7 +5,7 @@
 	</PropertyGroup>
 	<ItemGroup>
 		<PackageReference Include="Microsoft.Azure.DurableTask.Netherite.AzureFunctions" Version="1.3.2" />
-		<PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask" Version="2.12.0" />
+		<PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask" Version="2.13.0" />
```

Comment:
Ok, maybe that means we should not include this PR in the 1.5.0 release? 

Since we probably want this out sooner than later to make progress on the "partition corruption" theme.

Reply:
Also, we probably want the fix in https://github.com/Azure/azure-functions-durable-extension/pull/2748 to be included so I think it makes sense to wait for the next DF Extension release.

Reply:
> Ok, maybe that means we should not include this PR in the 1.5.0 release?

Yeah I'd say let's hold off, but prioritize this release shortly after.